### PR TITLE
chore: set up CI on v1 branch

### PR DESCRIPTION
**Pull request by busunkim96**
Set up GitHub Actions, update release script on v1

### CODE REVIEW

**Comment by parthea** [comment]
Please see commit [here](https://github.com/googleapis/python-vision/pull/324/commits/d59f3bcc953d3721d542acef6af3c3f8516bcb12) needed to fix docs build

**Comment by busunkim96** [comment]
@parthea The GH Actions docs build is passing.

**Comment by parthea** [comment]
This is the same issue as others where fail on missing interpreters isn't set.

File: noxfile.py [hunk 1]
```diff
@@ -145,7 +150,7 @@ def docs(session):
     shutil.rmtree(os.path.join("docs", "_build"), ignore_errors=True)
     session.run(
         "sphinx-build",
-        "-W",  # warnings as errors
+        # "-W",
```
Comment (busunkim96) [inline noxfile.py:153]:
```
copying static files... done
copying extra files... done
dumping search index in English (code: en)... done
dumping object inventory... done
build succeeded, 1043 warnings.
```

For whatever reason, the docs build has 1043 warnings, and I don't have the time to go through and fix each of these one by one. 

I tried changing the sphinx version, but that just resulted in a broken build.

Comment (busunkim96) [reply noxfile.py:153]: Turns out the error on the lower sphinx version was from https://github.com/sphinx-doc/sphinx/issues/10291, using a lower version of Jinja2 fixes it.

**Comment by busunkim96** [comment]
Docs issues have been resolved!

**Review by parthea — APPROVED** [review]

**Pull request merged**